core/templates/domain/skill/skill-creation-backend-api.service.spec.ts

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit test for SkillCreationBackendApiService.
+ */
+
+import { HttpErrorResponse } from '@angular/common/http';
+import { HttpClientTestingModule, HttpTestingController }
+  from '@angular/common/http/testing';
+import { TestBed, fakeAsync, flushMicrotasks } from '@angular/core/testing';
+
+import { CsrfTokenService } from 'services/csrf-token.service';
+import { SkillCreationBackendApiService, IRubricBackend,
+  ISkillCreationBackend } from
+  'domain/skill/skill-creation-backend-api.service';
+
+describe('Topic creation backend api service', () => {
+  let csrfService: CsrfTokenService = null;
+  let httpTestingController: HttpTestingController = null;
+  let skillCreationBackendApiService: SkillCreationBackendApiService = null;
+  let rubricDict: IRubricBackend = null;
+  let postData: ISkillCreationBackend = null;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule]
+    });
+
+    csrfService = TestBed.get(CsrfTokenService);
+    httpTestingController = TestBed.get(HttpTestingController);
+    skillCreationBackendApiService = TestBed.get(
+      SkillCreationBackendApiService);
+    spyOn(csrfService, 'getTokenAsync').and.callFake(() => {
```

Comment:
You can use `returnValue` instead of `callFake`

Reply:
changed the return value to returnValue